.github/workflows/coverity.yml

```diff
@@ -29,4 +30,4 @@ jobs:
           tar czf netplan.tar.gz cov-int
       - name: Upload results
         run: |
-          curl --form token=${{ secrets.COVERITY_TOKEN }} --form email=${{ secrets.COVERITY_EMAIL }} --form file=@netplan.tar.gz --form version="0.106" --form description="Coverity scan" https://scan.coverity.com/builds?project=Netplan
+          curl --form token=${{ secrets.COVERITY_TOKEN }} --form email=${{ secrets.COVERITY_EMAIL }} --form file=@netplan.tar.gz --form version="$(meson introspect coveritybuild --projectinfo | jq -r .version)" --form description="Coverity scan" https://scan.coverity.com/builds?project=Netplan
```

Comment:
thought (non-blocking): I wonder if we can get a more specific version for those coverity scans, using the git revision?

Something along the lines of this?
```bash
TAG=$(git describe --tags $(git rev-list --tags --max-count=1))  # find latest (stable) tag
REV=$(git rev-parse --short HEAD)  # get current git revision
VER="$TAG+git~$REV"
```

Reply:
sounds like a good idea